prod/.env.kitware-production.template

```diff
@@ -1,9 +1,13 @@
 # Django
 DJANGO_SETTINGS_MODULE=bats_ai.settings.kitware_production
 DJANGO_DATABASE_URL=postgres://postgres:postgres@postgres:5432/django
+DJANGO_DATABASE_NAME=django
+DJANGO_DATABASE_PASSWORD=postgres
 DJANGO_CELERY_BROKER_URL=amqp://rabbitmq:5672/
 DJANGO_MINIO_STORAGE_URL=http://minioAccessKey:minioSecretKey@minio:9000/django-storage
 DJANGO_SECRET_KEY=changeme
+DJANGO_MINIO_STORAGE_ACCESS_KEY=minioAccessKey
+DJANGO_MINIO_STORAGE_SECRET_KEY=minioSecretKey
```

Comment:
So these are needed specifically to make sure that the service we spin up has a default username/password? As far as Django settings go, all we need is the URI. maybe we should rename these so it's clear these settings aren't necessarily for Django.

Reply:
Yeah we can do this for the MINIO ones as well as the Postgres ones